### ISSUE TITLE
NTR: Tzanck smear: multinucleated giant cells (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8770]
New term request Tzanck smear: multinucleated giant cells (seidel_190501153742): Herpes virus; HSV; VZV (varicella zoster virus)  UMLS: 2171230

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5655652/

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000800
- label: Tzanck smear with multinucleated giant cells
- parents: [Positive skin infectious agent test (HP:0430117)](https://hpo.jax.org/app/browse/term/HP:0430117)
- pmid: PMID:29118470
- def: Tzanck smear demonstrates multinucleated giant cells, a sign that is suggestive for herpetic infections. Such smears are taken from skin lesions and display multinucleated cells with ground glass nuclei, nuclear moulding, and margination of nuclear chromatin typical of herpes simplex and varicella infection.
- comment: Tzanck smear was first introduced in 1947 by Frenchman Tzanck as a tool of diagnostic cytology for diagnosis of vesiculobullous conditions, especially herpes simplex. With time it evolved and Tzanck smear findings for several other dermatological conditions like immunobullous disorders, genodermatosis, cutaneous infections, and cutaneous tumors have also been described.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8770
- synonyms:
